processes/security_team_onboarding.md

```diff
@@ -0,0 +1,39 @@
+# On-boarding New Security WG Members
+
+This document aims to provide a clear process for on-boarding new team members to the Node.js Security WG.
+
+## Introducing new members
+New members may be nominated to join the WG, where they share an affinity to web or software security research, or activism around security and the Node.js project. All of which make good criteria to join the team but are not a closed list.
```

Comment:
What about self-nomination?

Reply:
It would also be good to clarify who is responsible for finding a buddy for a new WG member.

Reply:
I think that should come from the WG team through the nomination/join process